lib/exceptions.rb

```diff
@@ -8,7 +8,6 @@ class HostValidationError < ValidationError; end
   class LengthValidationError < ValidationError; end
   class DimensionsValidationError < ValidationError; end
   class StreamValidationError < ValidationError; end
-  class FilterValidationError < ValidationError; end
```

Comment:
From CI run - looks like spec still references this error class ... should be updated to use `Date::Error`?

Separately -- could we add some example date values which previously triggered errors but with this change now do not trigger errors to those specs to avoid future regression?